blossom/templates/website/generic_form.html

```diff
@@ -136,9 +125,9 @@ <h4>{{ subheader }}</h4>
     </div>
 
     <script>
-        document.addEventListener("DOMContentLoaded", function (event) {
-            const inputs = document.querySelectorAll('.inputfile');
-            Array.prototype.forEach.call(inputs, function (input) {
+        document.addEventListener("DOMContentLoaded", function () {
+            const inputs = Array.from(document.getElementsByClassName('inputfile'));
```

Comment:
Is this more efficient than `document.querySelectorAll('.inputfile')`?

I'm mostly interested in consistency with the other `querySelectorAll` refs in this code.

Reply:
That was my asumption, because the query doesn't have to get parsed. But I don't actually have any data on this

Reply:
So regarding the consistency, I tried to use the specific functions wherever possible.

Regarding performance I found this really old article: https://humanwhocodes.com/blog/2010/09/28/why-is-getelementsbytagname-faster-that-queryselectorall/

So in principal the query selector seems to be much faster because it returns a static element list instead of a live element collection, but I assume I negate that performance gain by using `Array.from`

Reply:
Ok I just changed it back to the query selectors. Being able to use `forEach` on the result directly is indeed much cleaner, I wasn't aware that it returned a different type of collection.